apps/emqx_conf/src/emqx_cluster_rpc.erl

```diff
@@ -31,10 +31,16 @@
     fast_forward_to_commit/2
 ]).
 -export([
-    get_node_tnx_id/1,
+    commit/2,
+    commit_status_trans/2,
     get_cluster_tnx_id/0,
+    get_node_tnx_id/1,
+    init_mfa/2,
     latest_tnx_id/0,
-    make_initiate_call_req/3
+    make_initiate_call_req/3,
+    read_next_mfa/1,
+    trans_query/1,
+    trans_status/0
```

Comment:
Should we separate the RPC-only exports into their own export block?

Reply:
In this partical case there was already an export block where some transaction funs were listed. I am not sure why, but I added the rest of the functions to the same block.